resources/ts/Components/Dropdown.tsx

```
@@ -1,94 +1,123 @@
+  setOpen: Dispatch<SetStateAction<boolean>>;
+  toggleOpen: () => void;
 }>({
-    open: false,
-    setOpen: () => {},
-    toggleOpen: () => {},
+  open: false,
+  setOpen: () => {},
+  toggleOpen: () => {},
 });
 
 const Dropdown = ({ children }: PropsWithChildren) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)